stress/common/src/main/java/alluxio/stress/jobservice/JobServiceMaxThroughputSummary.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.stress.jobservice;
+
+import alluxio.collections.Pair;
+import alluxio.stress.Parameters;
+import alluxio.stress.Summary;
+import alluxio.stress.graph.BarGraph;
+import alluxio.stress.graph.Graph;
+
+import com.google.common.base.Splitter;
+
+import java.text.DateFormat;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * The summary for the max throughput test.
+ */
+public final class JobServiceMaxThroughputSummary implements Summary {
```

Comment:
could you describe at a high level how this one is different from MaxThroughputSummary class and the rationale for having two separate classes?  would it be possible to have a templated class?

Reply:
I thought we can put refactoring as another PR but I can do it within this PR as well.